### ISSUE TITLE
missing ARIA alert role on login error message div

### ISSUE THREAD

**Issue by jmartin-sul** [LD4P/sinopia_editor#2376]
**Describe the bug**
> What went wrong? What did you expect to happen?

when a user attempts to log in with incorrect credentials, the error message `div` does not have an [ARIA alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role).

> - What is the impact on your work? (can't work / work stoppage; workaround exists; minor issue)

this could hinder interaction for users who use adaptive technology like screen readers.

> - What environment are you in? (stage or production)

discovered while writing feature tests, see https://github.com/LD4P/sinopia_editor/pull/2352#discussion_r474858506

> - What OS and browser are you using? (Mac/Windows; Safari, Firefox, etc.)

have not tested with an actual screen reader, just feature tests

> - If you're cataloging, what Resource Template were you using (name or ID)

n/a

**Please include a screenshot**

![Screen Shot 2020-08-26 at 12 10 54 PM](https://user-images.githubusercontent.com/7741604/91346376-99851300-e795-11ea-88ce-e21062b8bcec.png)

even after the above error message is displayed, no page elements have the `alert` role.
